docs/tutorials/AddRemark.md

```diff
@@ -284,29 +277,24 @@ Unfortunately, a change to `Question` will cause other commands to break, you wi
 
 </div>
 
-Refer to [this commit](https://github.com/se-edu/addressbook-level3/commit/ce998c37e65b92d35c91d28c7822cd139c2c0a5c) and check that you have got everything in order!
-
 
 ## Updating Storage
 
-AddressBook stores data by serializing `JsonAdaptedQuestion` into `json` with the help of an external library — Jackson. Let’s update `JsonAdaptedQuestion` to work with our new `Question`!
+SmartNus stores data by serializing `JsonAdaptedQuestion` into `json` with the help of an external library — Jackson. Let’s update `JsonAdaptedQuestion` to work with our new `Question`!
 
 While the changes to code may be minimal, the test data will have to be updated as well.
 
 <div markdown="span" class="alert alert-warning">
 
-:exclamation: You must delete AddressBook’s storage file located at `/data/addressbook.json` before running it! Not doing so will cause AddressBook to default to an empty address book!
+:exclamation: You must delete SmartNus' storage file located at `/data/smartnus.json` before running it! Not doing so will cause SmartNus to default to an empty questin list!
```

Comment:
Typo.
Correct line: 
:exclamation: You must delete SmartNus' storage file located at `/data/smartnus.json` before running it! Not doing so will cause SmartNus to default to an empty question list!